tests/gitea-repositories-meta/user2/lfs.git/description

```diff
@@ -0,0 +1 @@
+Unnamed repository; edit this file 'description' to name the repository.
```

Comment:
Maybe this file doesn't need to be committed.

Reply:
Instead, I've given it a useful description like lunny did in https://github.com/go-gitea/gitea/commit/36a2d2f919ee445f0374c210309b336e7c3c1f8e#diff-323c2a691abdc548ffd31227cd89b2420b60efbff2bfd7a17ad2b53ec199c08a